examples/tutorial_notebook.ipynb

```diff
@@ -273,6 +273,45 @@
     "print(f\"Experiment last update date: {experiment.updated_at}\")\n",
     "print(f\"Experiment tags: {experiment.tags}\")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Search through Experiments"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from datetime import datetime, timedelta\n",
+    "\n",
+    "# Search through the experiments by their title or alias. The search is wild card type, so it returns any combination of the provider string like *search_string*.\n",
```

Comment:
```suggestion
    "# Search through the experiments by their title or experiment ID. It will return all experiments where search string is a substring of the title or the experiment ID\n",
```

If I'm not mistaken, this is what happens:

```
                orm.Experiment.title.ilike(f"%{title_filter}%"),
                orm.Experiment.alias.ilike(f"%{title_filter}%"),
```

1. I think that "wildcard" word may mislead users and they will start writing things like `experiment-202001*-xyz`.
2. The way how search is implemented will probably mess with `%` (which is bearable) and what is more confusing, with `_` symbol - if will return more than expected, and `[]`, `{}`. https://www.w3schools.com/sql/sql_wildcards.asp